### ISSUE TITLE
[4.3.0] Trailing comma when checking the line length limit.

### ISSUE THREAD

**Issue by felixxm** [timothycrosley/isort#654]
Version `4.3.0` doesn't take into account a trailing comma when checking the line length limit e.g. (
`include_trailing_comma = true`, `line_length = 79`):
```diff
 from django.contrib.gis.gdal.field import (
-    OFTDate, OFTDateTime, OFTInteger, OFTInteger64, OFTReal, OFTString,
-    OFTTime,
+    OFTDate, OFTDateTime, OFTInteger, OFTInteger64, OFTReal, OFTString, OFTTime,
 )
```

**Comment by timothycrosley** [comment]
This should now be fixed in release 4.3.1, Thank you!

**Comment by felixxm** [comment]
Thanks! 👍
